### PR TITLE
Addressing Passport Model

### PR DESCRIPTION
**Pull request by hannestschofenig**
Addressing https://github.com/lamps-wg/csr-attestation/issues/37

### CODE REVIEW

File: draft-ietf-lamps-csr-attestation.md [hunk 1]
```diff
@@ -146,11 +146,21 @@ message interchangeably.
...
 
-{{fig-arch}} shows the high-level communication pattern of the RATS passport
-model where the Attester transmits the Evidence in the CSR to the RA
-and the CA. The Verifier processes the received Evidence and computes
-an Attestation Result, which is then processed by the RA/CA prior to the
-certificate issuance.
+{{fig-arch}} shows the high-level communication pattern of the RATS
+background check model where the Attester transmits the Evidence in the
+CSR to the RA and the CA, which is subsequently forwarded to the Verifier.
+The Verifier processes the received Evidence and computes an Attestation
```
Comment (henkbirkholz) [inline draft-ietf-lamps-csr-attestation.md:152]:
```suggestion
The Verifier appraises the received Evidence and computes an Attestation
```

**Comment by henkbirkholz** [comment]
lgtm! just one nit

**Review by henkbirkholz — APPROVED** [review]
:+1:

**Review by ounsworth — APPROVED** [review]

**Comment by ounsworth** [comment]
Looks good to me too.

**Pull request merged**